### PR TITLE
Create complete list of all press

### PR DESCRIPTION
**Pull request by ElenaFdR**
This is the full list I created for [publiccode.net PR 178](https://github.com/publiccodenet/publiccode.net/pull/178) - except I know it's not full! But collating what's here seems like a good starting point, and lets me get creative with the publiccode.net layout.

-----
[View rendered activities/value-and-impact/all-press.md](https://github.com/publiccodenet/about/blob/press-impact/activities/value-and-impact/all-press.md)
[View rendered activities/value-and-impact/index.md](https://github.com/publiccodenet/about/blob/press-impact/activities/value-and-impact/index.md)

### CODE REVIEW

**Comment by Ainali** [comment]
This is a great start and will be a very useful archive for us!

**Comment by ElenaFdR** [comment]
Re not full, I believe we're missing some non-English references. For example, I believe OS2 has written about us more often than just the interview I have listed.

**Comment by Ainali** [comment]
Question: should we provide an English translation of the links in other languages?

**Comment by ElenaFdR** [comment]
1. I think translating the titles for this list is courteous and helps readers decide whether to click a link.
2. Re full articles, which format would you put this in? Where would you host it? Do we need to credit the (free) translation software we use?

**Comment by Ainali** [comment]
> 1. I think translating the titles for this list is courteous and helps readers decide whether to click a link.
> 2. Re full articles, which format would you put this in? Where would you host it? Do we need to credit the (free) translation software we use?

I was only thinking of 1, the title. The full articles is not only a lot of work, but also need permission (unless the article is freely licensed) from the author.

**Comment by ElenaFdR** [comment]
That makes much more sense!

**Comment by Ainali** [comment]
We have two titles in Dutch now, then this should be ready to be merged.

File: activities/value-and-impact/all-press.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+type: resource
+---
+
+
+# All press and media mentions of the Foundation for Public Code
+
+Here's the complete list of all mentions of the Foundation for Public Code on 3rd party platforms.
```
Comment (Ainali) [inline activities/value-and-impact/all-press.md:8]: "third" would be easier to read. Is there a reason for using numbers here?

File: activities/value-and-impact/all-press.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+We're always happy to work with journalists  - just email us at <press@publiccode.net>.
+
+You may also be interested in:
+
+- [a list of policy documents that mention the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/policy-documents.html)
+- [our conference talks and articles we've written](https://projects.publiccode.net/talks-and-articles.html)
+
+## 2020
+
+- [Help: open source software will be mandatory, what now?](https://opensource.pleio.nl/news/view/7155b323-98f4-4ca0-9777-7a53ffbf2d16/help-open-source-software-wordt-verplicht-wat-nu), summary of a panel discussion with Boris van Hoytema, Ivonne Janssen-Dings and Koos Steenbergen at the Conferentie Nederland Digitaal in February 2021 (in Dutch)
```
Comment (Ainali) [inline activities/value-and-impact/all-press.md:19]: In this bullet point list there sometimes are a comma after the title and sometimes not. This should be consistent (but I don't know if we want them or not).

File: activities/value-and-impact/all-press.md (hunk 1 above)
Comment (ElenaFdR) [reply activities/value-and-impact/all-press.md:8]: Nope! Happy to change.

File: activities/value-and-impact/all-press.md (hunk 2 above)
Comment (ElenaFdR) [reply activities/value-and-impact/all-press.md:19]: Also dashes! I'll standardize.

Comment (bvhme) [reply activities/value-and-impact/all-press.md:19]: Also, this is not in 2020

**Review by Ainali — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

**Pull request merged**